ci/constants.py

```diff
@@ -96,9 +98,13 @@ class TargetPython(Enum):
     'sympy',
     'vlc',
 ])
+# to be created via https://github.com/kivy/python-for-android/issues/XXXX
```

Comment:
I think that we should create an issue page for python2.
What do you think @AndreMiras ?

Reply:
I don't know if it's worth the effort, but feel free if want